samsungtvws/async_rest.py

```diff
@@ -32,18 +32,21 @@ def __init__(
             timeout=timeout,
         )
         self.session = session
+        self.client_timeout = aiohttp.ClientTimeout(timeout)
```

Comment:
Do we need to expose it?
```suggestion
        self._client_timeout = aiohttp.ClientTimeout(timeout)
```